tests/cases/fourslash/quickInfoJsDocTags12.ts

```diff
@@ -0,0 +1,17 @@
+/// <reference path="fourslash.ts" />
```

Comment:
Can you add a test showing what happens if there are more than one parameters with the same name? E.g. `function foo(a, { a })`